docs/CLI-Reference.md

```diff
@@ -21,6 +21,7 @@ This file is auto-generated from the output of `cl help -v` and provides the lis
           -p, --pack               If path is an archive file (e.g., zip, tar.gz), keep it packed.
           -z, --force-compression  Always use compression (this may speed up single-file uploads over a slow network).
           -w, --worksheet-spec     Upload to this worksheet ([(<alias>|<address>)::](<uuid>|<name>)).
+          -i, --ignore             Name of file containing patterns matching files and directories to exclude from upload. This option is currently only supported with the GNU tar library.
```

Comment:
@percyliang  Do you know why these files were generated when I ran the precommit script? I don't see anything in the diff that explains them, perhaps I can try a rebase.

Reply:
precommit.sh auto generates some docs, the current master branch should have this change already so rebasing should do it